GLAA.Services/PublicRegister/PublicRegisterViewModelBuilder.cs

```diff
@@ -123,7 +125,8 @@ private PublicRegisterLicenceDetailViewModel BuildDetail(Licence licence)
         {
             var publicRegisterLicenceDetailViewModel = _mapper.Map<PublicRegisterLicenceDetailViewModel>(licence);
 
-            publicRegisterLicenceDetailViewModel.Address.Countries = _referenceDataProvider.GetCountries();
+            if (publicRegisterLicenceDetailViewModel.Address != null)
```

Comment:
Could the view model ensue the existence of the `Address` property / object in the `ctor`?

Reply:
I think in this particular instance, this is done so that it will only populate the countries list if we need to (i.e. it only needs countries if the address is not null).  Again though, this implementation will likely change.